SonarQube.TeamBuild.PreProcessor/PreprocessorObjectFactory.cs

```diff
@@ -45,7 +45,7 @@ public ISonarQubeServer CreateSonarQubeServer(ProcessedArgs args, ILogger logger
             string password = args.GetSetting(SonarProperties.SonarPassword, null);
             string hostUrl = args.GetSetting(SonarProperties.HostUrl, null);
 
-            this.server = new SonarWebService(new WebClientDownloader(username, password), hostUrl, logger);
+            this.server = new SonarWebService(new WebClientDownloader(username, password, logger), hostUrl, logger);
```

Comment:
Miscellaneous: I've added debug logging to the three methods in the SonarWebService. There have been a number of support questions when it would have been useful to check the urls; all urls will now be logged in debug mode.